pkg/azure/client/loadbalancers.go

```diff
@@ -25,6 +25,15 @@ func NewLoadBalancersClient(auth internal.ClientAuth, tc azcore.TokenCredential,
 	return &LoadBalancersClient{client}, err
 }
 
+// Get gets a given virtual load balancer by name
+func (c *LoadBalancersClient) Get(ctx context.Context, resourceGroupName, name string) (*armnetwork.LoadBalancer, error) {
+	res, err := c.client.Get(ctx, resourceGroupName, name, nil)
+	if err != nil {
+		return nil, FilterNotFoundError(err)
```

Comment:
Haven't checked around, but is this how we usually do it? This get method will always shadow notFound, I think this should be done when calling this function (if the caller wants it)

Reply:
This is also how the other client implementations work. The comment is fair, but for local consistency it would be applied in a separate PR.